client/settings/express-checkout-settings/woopay-settings.js

```diff
@@ -226,10 +226,17 @@ const WooPaySettings = ( { section } ) => {
 									onChange={
 										updateIsWooPayGlobalThemeSupportEnabled
 									}
-									label={ __(
-										'Enable global theme support',
-										'woocommerce-payments'
-									) }
+									label={
+										<div className="woopay-settings__global-theme-label">
```

Comment:
💭 the manual styling you've added here works well, no need to change it, especially since it is temporary.

I just wanted to mention that the WooPayments [`Chip` component](https://github.com/Automattic/woocommerce-payments/tree/develop/client/components/chip) might be a good fit here.

![image](https://github.com/user-attachments/assets/0d3d76c5-1af9-4cad-a647-3e8c7225444d)

Related: the new `Badge` component coming to `@wordpress/components` (https://github.com/Automattic/woocommerce-payments/pull/8382#issuecomment-2531120875) should replace the `Chip` component, but as you point out, we'll have to upgrade to the latest/next version of the library to do so.

Reply:
Good to learn about the `Chip` component. Thanks @Jinksi 
It does seem like a good fit here. Since this change is temporary, I'll leave the current implementation as is.